docs/framework/react/reference/usePrefetchQuery.md

```diff
@@ -0,0 +1,24 @@
+---
+id: usePrefetchQuery
+title: usePrefetchQuery
+---
+
+```tsx
+const result = usePrefetchQuery(options)
+```
+
+**Options**
+
+You can pass everything to `usePrefetchQuery` that you can pass to [`queryOptions`](../queryOptions) or [`useQuery`](../useQuery). Some options will have no effect because, under the hood, `usePrefetchQuery` uses `queryClient.prefetchQuery`. Since the options are general, TypeScript will not warn you about any excess properties.
```

Comment:
"some options will have no effect": This is a good point actually. I think we shouldn't allow to pass in `UseQueryOptions` but rather `FetchQueryOptions`

then we will also know what we have to type-test: for example, that the `refetchInterval` option cannot be passed.

Reply:
Done \O/